### ISSUE TITLE
Issue with .ts files using import statements and registering commands

### ISSUE THREAD

**Issue by Conman1161** [Snazzah/slash-create#205]
I updated Slash Create to version 5.0.0 and I am not sure if this is an issue related to that version or not, but I was not having this issue prior to 5.0.0

Tl;DR: The `registerCommandsIn` function causes issues with TypeScript files that use `import` statements.

When using ```Creator.registerCommandsIn(`${__dirname}/commands/`);``` and a `.ts` file with an `import` for modules is attempted to be registered, I get the following error:
```
Exception has occurred: c:\DiscordBots\JohnathanDazzle\commands\dice\average.ts:1
import { SlashCommand, CommandOptionType, SlashCreator, CommandContext } from 'slash-create';
^^^^^^

SyntaxError: Cannot use import statement outside a module
```

I think it's because of `require` but I am not familiar enough with how TS works to be certain. Below I have the original block and my bodge to address the issue, but I doubt that it's the best solution for the issue at hand.

https://github.com/Snazzah/slash-create/blob/784d1c44f37f84a5e4e61b6bb6c5e00d5c9e496e/src/creator.ts#L172-L184

```ts
registerCommandsIn(commandPath) {
    const paths = util_1.getFiles(commandPath);
    const commands = [];
    for (const filePath of paths) {
        try {
            if (filePath.substring(filePath.length - 3) === '.ts') { // "fix" here
                console.log(`ts, ignoring...`);
                continue;
            }
            commands.push(require(filePath));
        }
        catch (e) {
            this.emit('error', new Error(`Failed to load command ${filePath}: ${e}`));
            throw e;
        }
    }
    return this.registerCommands(commands, true);
}
```

**Comment by Snazzah** [comment]
That's weird since  `require-all` pretty much did the same thing. Did you make any changes to your tsconfig.json?

**Comment by Conman1161** [comment]
I haven't made any changes to my tsconfig since updating to 5.0.0

Here's a link in case i'm not aware if a bad setting https://github.com/Conman1161/johnathan-dazzle/blob/main/tsconfig.json

**Comment by Snazzah** [comment]
Found the issue, fixed in [v5.0.1](https://github.com/Snazzah/slash-create/releases/tag/v5.0.1).
